src/Compilers/CSharp/Test/Syntax/LexicalAndXml/LexicalErrorTests.cs

```diff
@@ -963,17 +963,17 @@ public static void Main()
 ";
 
             ParserErrorMessageTests.ParseAndValidate(test,
```

Comment:
these tests changed because they contain trivia that used to improperly be associated with the interpolation expression (as leading trivia).  Once the trivia moved, teh location of the error reported changed to a different place.